docs/v3.x/getting-started/nuxt-js.md

```diff
@@ -0,0 +1,755 @@
+import axios from 'axios'
+
+export default {
+  name: 'App',
+  data () {
+    return {
+      restaurants: [],
+      error: null
+    }
+  },
+  mounted () {
+    fetch("http://localhost:1337/restaurants", {
+      headers: {
+         'Content-Type': 'application/json'
+      },
+    }).then(response => response.json())
+      .then(data => {
+        this.restaurants = data
+      })
+      .catch((error) => {
+        this.error = error
+      });
+  }
+}
+</script>
+```
+
+:::
+
+::::
+
+### POST Request your collection type
+
+Execute a POST request on the `restaurant` Collection Type in order to create a restaurant.
+
+Be sure that you activated the `create` permission for the `restaurant` Collection Type and the `find` permission for the `category` Collection type.
+
+
+:::: tabs
+
+::: tab @nuxtjs/strapi
+
+```js
+try {
+  await this.$strapi.create('restaurants', {
+    name: 'Dolemon Sushi',
+    description: 'Unmissable Japanese Sushi restaurant. The cheese and salmon makis are delicious'
+  })
+} catch (error) {
+  this.error = error
+}
+```
+:::
+
+::: tab axios
+
+*Request*
+
+```js
+axios.post('http://localhost:1337/restaurants', {
+    name: 'Dolemon Sushi',
+    description: 'Unmissable Japanese Sushi restaurant. The cheese and salmon makis are delicious'
+  })
+  .then(response => {
+    console.log(response);
+  })
+  .catch(error => {
+    console.log(error);
+  });
+```
+:::
+
+::: tab fetch
+
+*Request*
+
+```js
+fetch('http://localhost:1337/restaurants', {
+   method: 'POST',
+   headers: {
+      'Content-Type': 'application/json'
+   },
+   body: JSON.stringify({
+     name: 'Dolemon Sushi',
+     description: 'Unmissable Japanese Sushi restaurant. The cheese and salmon makis are delicious'
+   })
+ }).then(response => {
+   return response.json();
+ }).then(data => {
+   console.log(data);
+ }).catch(error => {
+   console.error(error);
+ });
+```
+:::
+
+*Response*
+
+```json
+{
+    "id": 2,
+    "name": "Dolemon Sushi",
+    "description": "Unmissable Japanese Sushi restaurant. The cheese and salmon makis are delicious",
+    "created_by": null,
+    "updated_by": null,
+    "created_at": "2020-08-04T09:57:11.669Z",
+    "updated_at": "2020-08-04T09:57:11.669Z",
+    "categories": [
+    ]
+}
+```
+
+::::
+
+### Example
+
+:::: tabs
+
+::: tab @nuxtjs/strapi
+
+`./pages/index.vue`
+
+```js
+<template>
+<div class="container">
+  <div v-if="error">
+    {{ error }}
+  </div>
+
+  <form id="form" v-on:submit="handleSubmit" v-else>
+    <h3>Restaurants</h3>
+    <br>
+
+    <label for="name">Name</label>
+    <input id="name" v-model="name" type="text" name="name">
+
+    <label for="description">Description</label>
+    <input id="description" v-model="description" type="text" name="description">
+    <div>
+      <br />
+      <b>Select categories</b>
+      <br>
+      <div v-for="category in allCategories" :key="category.id">
+        <label>{{ category.name }}</label>
+        <input
+          type="checkbox"
```

Comment:
Are you sure you have the same behaviour for the checkboxes as in your next app?